contracts/SignerBondsUniswapV2.sol

```diff
@@ -93,8 +96,23 @@ contract SignerBondsUniswapV2 is ISignerBondsSwapStrategy, Ownable {
     // If false, open auctions are not taken into account.
     bool public revertIfAuctionOpen = true;
 
+    event SignerBondsSwapOperatorApproved(address operator, uint256 timestamp);
```

Comment:
Do we need to add `timestamp` parameter to these events? The timestamp can be read from block add we add `timestamp` only to `Begin*` governance events just because the timestamp is the part of the business logic and not a metadata - that is, when the governance delay started.

Reply:
I removed the timestamps in https://github.com/keep-network/coverage-pools/pull/148/commits/cee62f0dba2afa4442e29eff73ffec2170ac3dd2.